app/presenters/result_set_presenter.rb

```diff
@@ -114,4 +121,63 @@ def sort_option
 
     sort_option
   end
+
+  def tagged_to_all?(facet_key, metadata)
```

Comment:
I couldn't find where `tagged_to_all` and some of the other new methods on this class are used. Should these be removed?